internal/service/elasticache/replication_group_test.go

```diff
@@ -66,6 +66,102 @@ func TestAccElastiCacheReplicationGroup_basic(t *testing.T) {
 	})
 }
 
+func TestAccElastiCacheReplicationGroup_network_type_ipv6(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping long-running test in short mode")
+	}
+
+	var rg elasticache.ReplicationGroup
+	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
+	resourceName := "aws_elasticache_replication_group.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ErrorCheck:               acctest.ErrorCheck(t, elasticache.EndpointsID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckReplicationGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccReplicationGroupConfig_basic(rName),
+				Check: resource.ComposeAggregateTestCheckFunc(
```

Comment:
Get rid of some of the less interesting attributes down here, just focus on what's mandatory to check + what the test covers.

Reply:
Completed